values.yaml

```diff
@@ -127,6 +127,11 @@ server:
   #     memory: 256Mi
   #     cpu: 250m
 
+  # Pod's group id.
+  gid: 1000
+  # Pod's user id.
+  uid: 1000
```

Comment:
This should be `100`.  The servers UID and GID by default are `100:1000`.

Reply:
Fixed.